### ISSUE TITLE
Bench2Drive dataset organization

### ISSUE THREAD

**Issue by poodarchu** [RenzKa/simlingo#29]
hello, my Bench2Drive dataset seems to be different from yours:

$ tree -d -L 3
.
├── AccidentTwoWays_Town12_Route1102_Weather10
│   ├── anno
│   ├── camera
│   │   ├── depth_back
│   │   ├── depth_back_left
│   │   ├── depth_back_right
│   │   ├── depth_front
│   │   ├── depth_front_left
│   │   ├── depth_front_right
│   │   ├── instance_back
│   │   ├── instance_back_left
│   │   ├── instance_back_right
│   │   ├── instance_front
│   │   ├── instance_front_left
│   │   ├── instance_front_right
│   │   ├── rgb_back
│   │   ├── rgb_back_left
│   │   ├── rgb_back_right
│   │   ├── rgb_front
│   │   ├── rgb_front_left
│   │   ├── rgb_front_right
│   │   ├── rgb_top_down
│   │   ├── semantic_back
│   │   ├── semantic_back_left
│   │   ├── semantic_back_right
│   │   ├── semantic_front
│   │   ├── semantic_front_left
│   │   └── semantic_front_right
│   ├── expert_assessment
│   ├── lidar
│   └── radar

I cannot find the xml files, thus cannot conduct evaluation.

**Comment by poodarchu** [comment]
Evaluation
The model file can be downloaded from huggingface: https://huggingface.co/RenzKa/simlingo.

Bench2Drive
Bench2Drive is a CARLA benchmark proposed by the paper [Bench2Drive: Towards Multi-Ability Benchmarking of Closed-Loop End-To-End Autonomous Driving](https://arxiv.org/abs/2406.03877). It consists of 220 very short (~150m) routes split across all towns with 1 safety critical scenario in each route. Since it uses all towns for training, the methods have seen the test towns during training, so it can be considered a 'training' benchmark (reminiscent of level 4 driving). The benchmark also comes with a training dataset generated by the [Think2Drive](https://arxiv.org/abs/2402.16720) expert, but we use the open-source expert [PDM-Lite](https://arxiv.org/abs/2312.14150) that achieves better resuslts and can be adapted to collect the necessary labels to produce VQA, Commentary and Dreamer data. The benchmark and additional instructions can be found in the [Bench2Drive](https://github.com/RenzKa/simlingo/blob/main/Bench2Drive) folder.

Start eval: Evaluation on a SLURM cluster can be run with [start_eval_simlingo.py](https://github.com/RenzKa/simlingo/blob/main/start_eval_simlingo.py). The config dictionary needs to be adjusted with the correct names and paths. Most things that need to be chaged are marked with TODO tags in [start_eval_simlingo.py](https://github.com/RenzKa/simlingo/blob/main/start_eval_simlingo.py).

Get results: The script [Bench2Drive/tools/merge_route_json.py](https://github.com/RenzKa/simlingo/blob/main/Bench2Drive/tools/merge_route_json.py) can be used to obtain the final metrics after the evaluation is done. Make sure that all 220 routes are evaluated.

The Bench2Drive folder is based on version 0.0.3 of the [Bench2Drive repository](https://github.com/Thinklab-SJTU/Bench2Drive). Please cite the [Bench2Drive paper](https://arxiv.org/abs/2406.03877) when using the benchmark.

**Comment by poodarchu** [comment]
any hint on this? @RenzKa

**Comment by RenzKa** [comment]
Sorry for this. I uploaded the split bench2drive routes here: https://github.com/RenzKa/simlingo/tree/main/leaderboard/data/bench2drive_split
